fudge-ratio/src/main/webapp/script.js

```diff
@@ -85,3 +85,20 @@ const formatTaskString = () => {
 const displayMessage = () => {
     document.getElementById("finalMessage").innerHTML = "Next time please allocate" + duration + " for "  + formatTaskString();
 };
+
+function loadActivities() {
+    fetch('/data').then(response => response.json()).then((activities) => {
+        activities.forEach((activity) => {
+            var activityType = activity.type;
```

Comment:
My understanding is that var is outdated now because it's not block-scoped, so using let is better and common practice.